neo/SmartContract/Native/Votes/VoteContract.cs

```diff
@@ -0,0 +1,505 @@
+                {
+                    StorageKey key = CreateStorageKey(Prefix_AccessControl, id);
+                    StorageItem storage_Access = engine.Snapshot.Storages.TryGet(key);
+                    if (storage_Access is null)
+                    {
+                        return AddAccess(engine.Snapshot, key, newVoter);
+                    }
+                    else
+                    {
+                        storage_Access = engine.Snapshot.Storages.GetAndChange(key);
+                        HashSet<UInt160> oldVoter = ConvertBytesToUserArray(storage_Access.Value);
+                        newVoter.UnionWith(oldVoter);
+                        storage_Access.Value = ConvertUserArrayToBytes(newVoter);
+                        return true;
+                    }
+                }
+                catch
+                {
+                    return false;
+                }
+            }
+            else
+            {
+                try
+                {
+                    StorageKey key = CreateStorageKey(Prefix_AccessControl, id);
+                    StorageItem storage_Access = engine.Snapshot.Storages.TryGet(key);
+                    if (storage_Access is null)
+                    {
+                        return AddAccess(engine.Snapshot, key, newVoter);
+                    }
+                    else
+                    {
+                        storage_Access = engine.Snapshot.Storages.GetAndChange(key);
+                        storage_Access.Value = ConvertUserArrayToBytes(newVoter);
+                        return true;
+                    }
+                }
+                catch
+                {
+                    return false;
+                }
+            }
+        }
+        [ContractMethod(0_01000000, ContractParameterType.ByteArray, SafeMethod = true)]
+        private StackItem GetResult(ApplicationEngine engine, VMArray args)
+        {
+            if (args[0] == null) return false;
+            UInt256 TxHash = new UInt256(args[0].GetByteArray());
+            var id = new Crypto().Hash160(VoteCreateState.ConcatByte(TxHash.ToArray(), (engine.CallingScriptHash ?? this.Hash).ToArray()));
+
+            StorageKey key = CreateStorageKey(Prefix_Result, id);
+            StorageItem result = engine.Snapshot.Storages.TryGet(key);
+            if (result == null) return false;
+
+            return result.Value;
+        }
+
+
+        private bool RegisterVote(Snapshot snapshot, VoteCreateState createState)
+        {
+            StorageKey key = CreateStorageKey(Prefix_CreateVote, createState.GetId());
+
+            if (snapshot.Storages.TryGet(key) != null) return false;
+
+            try
+            {
+                snapshot.Storages.Add(key, new StorageItem
+                {
+                    Value = createState.ToArray()
+                });
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
```

Comment:
plz, refer the format of contract
https://github.com/neo-project/neo/blob/d9d54b5dfe44e41561ff729b08b9b7caf1f0d6b9/src/neo/SmartContract/InteropService.Contract.cs#L37-L43